src/__data__/inAppMessages.ts

```diff
@@ -1,165 +1,172 @@
 import { InAppMessage } from '../inapp/types';
 
```

Comment:
The changes to this file are minimal:
* Pulled each inapp message out to its own named variable for ease of knowing differences
* Updated "normal" message to not have expired (inevitably for now) timestamp
* Set up one that can be swapped out with the commented "previouslyCachedMessage" if a mock for the message delta call is needed